docs/sources/clients/promtail/configuration.md

```diff
@@ -705,6 +706,27 @@ tenant:
   [ value: <string> ]
 ```
 
+#### replace
+
+The replace stage is a parsing stage that parses a log line using
+a regular expression and replaces the log line.
+
+```yaml
+replace:
+  # The RE2 regular expression. Each named capture group will be added to extracted.
+  # Each capture group and named capture group will be replaced with the value given in `replace`
+  expression: <string>
```

Comment:
can we add an example for each one? (expression, source and replace). In the same document string you already written.

Reply:
The examples are already written here @kavirajk 
https://grafana.com/docs/loki/latest/clients/promtail/stages/replace/